runtime/doc/mbyte.txt

```diff
@@ -335,8 +335,8 @@ Vim will automatically convert from one to another encoding in several places:
   "utf-8" (requires a gettext version that supports this).
 - When reading a Vim script where |:scriptencoding| is different from
   "utf-8".
-Most of these require the |+iconv| feature.  Conversion for reading and
```

Comment:
`+iconv` is still a feature actually.

Reply:
Oh yeah my bad, I didn't spot that, thanks